tester.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+version="V0.0.3"
+usage="$(basename "$0") [-help] [-v] -- Simple test suite for dfshack
+
+where:
+    -h,   show this help text
+    -v,   Shows version information"
+
+###############################################################################
+#Function: auto_testing                                                       #
+#Prupose: Runs auto test case for dfshack by creating severle files changing  #
```

Comment:
"Purpose", "Several"

(If you're using vim, `:set spell` should turn on the spellchecker)